srcpkgs/alacritty/template

```diff
@@ -13,13 +13,17 @@ license="Apache-2.0"
 homepage="https://github.com/alacritty/alacritty"
 changelog="https://raw.githubusercontent.com/alacritty/alacritty/master/CHANGELOG.md"
 distfiles="${homepage}/archive/v${version}.tar.gz"
-checksum=6d3aaac9e0477f903563b6fb26e089118407cdbfe952a1e2ffbf4e971b7062b3
+checksum=5dee9c94f944b742b0189dd87c43c87175ffadde8049abb4668ca5a3e68fd65a
 
 case "$XBPS_TARGET_MACHINE" in
 	ppc64*) ;;
 	ppc*) broken="ftbfs: thread 'main' panicked at 'slice index starts at 4289555962 but ends at 114', /builddir/rustc-1.46.0-src/src/libstd/io/mod.rs:396:27";;
 esac
 
+case "$XBPS_TARGET_MACHINE" in
+	aarch64*|armv*) broken="ftbfs in serde_yaml dependency, already reported upstream";;
```

Comment:
Please link the actual issue here.

And I'd say it's reason enough to hold off on the update, unless this has particularly relevant new featuers?